protocol-designer/src/steplist/formLevel/handleFormChange/dependentFieldsUpdateMoveLiquid.ts

```diff
@@ -512,18 +513,29 @@ const updatePatchOnPipetteChannelChange = (
     const sourceLabware = labwareEntities[sourceLabwareId]
     const sourceLabwareDef = sourceLabware.def
     const destLabware = labwareEntities[destLabwareId]
-    const destLabwareDef = destLabware.def
+
+    // @ts-expect-error(sa, 2021-6-14): appliedPatch.pipette does not exist. Address in #3161
+    const pipetteId: string = appliedPatch.pipette
```

Comment:
does this work? I know there are a bunch of` ts-expect-errors` in this file but if this way works, i'd prefer to limit our usage of `ts-expect-errors` when possible. At some point we need to create the types for appliedPatch fully 😄 

```suggestion
    const pipetteId: string = appliedPatch.pipette as string
```

Reply:
yes this does work! should i also make this change on line 487?

Reply:
or i'll move it outside of the conditional statement as they both used `pipetteId`